### ISSUE TITLE
Globus Connect Server 5.1

### ISSUE THREAD

**Issue by akkornel** [stanford-rc/globus.stanford.edu#17]
Sometime next week (1st week of May), Globus Connect Server 5.1 should be coming out.  That's going to change the Server docs.

So, the Server docs will need to be updated (and possibly split) into common stuff, GCS 4 stuff, and GCS 5.1 stuff.

**Comment by akkornel** [comment]
This is going to be on hold for a while.  In particular, there are two things that GCS 5 really needs to implement before many of us can use it:

• Mapped POSIX Collections
• Multiple DTNs in one endpoint

After that, documentation work can start.  Once that is done, we can start pushing GCS 5 out to people who are using CILogon authentication.  But it's important to note, we won't be able to deprecate our existing GCS 4 stuff until GCS5 also supports MyProxy for mapped collections.
